autogen/cluster_regional.tf

```diff
@@ -36,6 +36,9 @@ resource "google_container_cluster" "primary" {
   logging_service    = "${var.logging_service}"
   monitoring_service = "${var.monitoring_service}"
 
+{% if private_cluster %}
```

Comment:
What is the motivation to only provide this for private clusters?

Reply:
Because of this change valid only for google-beta provider

Reply:
Got it. Thanks!